examples/multiple_test_pages/tests.js

```diff
@@ -0,0 +1,11 @@
+for (var i = 0; i < 200; i++) {
+  if (window.location.search === '?batch=1') {
```

Comment:
@Vanuan yes, that should still work and you will never have more then `-P` parallel running launchers.